src/com/goide/inspections/GoCgoInTestInspection.java

```diff
@@ -21,11 +21,14 @@
 import com.goide.quickfix.GoDeleteImportQuickFix;
 import com.goide.runconfig.testing.GoTestFinder;
 import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.openapi.progress.ProgressManager;
 import org.jetbrains.annotations.NotNull;
 
 public class GoCgoInTestInspection extends GoInspectionBase {
   @Override
   protected void checkFile(@NotNull GoFile file, @NotNull ProblemsHolder problemsHolder) {
+    super.checkFile(file, problemsHolder);
+    ProgressManager.checkCanceled();
```

Comment:
Oh, ok, I've missed that. If there's a guideline for how to add this I'd be happy to update this to be more useful.